### ISSUE TITLE
Crash On Login

### ISSUE THREAD

**Issue by Meus-Artis** [TechnicPack/TechnicLauncher#343]
Technic Launcher Error Report:
( Please submit this report to https://github.com/TechnicPack/TechnicLauncher/issues )
    Launcher Build: 358

```
Selected Pack: Tekkit Classic
```

Stack Trace:
    Exception: NoSuchElementException
    Message: null
    Trace:
        java.util.LinkedList.getFirst(Unknown Source)
        java.awt.DefaultKeyboardFocusManager.repostIfFollowsKeyEvents(Unknown Source)
        java.awt.DefaultKeyboardFocusManager.dispatchEvent(Unknown Source)
        java.awt.Component.dispatchEventImpl(Unknown Source)
        java.awt.Container.dispatchEventImpl(Unknown Source)
        java.awt.Window.dispatchEventImpl(Unknown Source)
        java.awt.Component.dispatchEvent(Unknown Source)
        java.awt.EventQueue.dispatchEventImpl(Unknown Source)
        java.awt.EventQueue.access$400(Unknown Source)
        java.awt.EventQueue$3.run(Unknown Source)
        java.awt.EventQueue$3.run(Unknown Source)
        java.security.AccessController.doPrivileged(Native Method)
        java.security.ProtectionDomain$1.doIntersectionPrivilege(Unknown Source)
        java.security.ProtectionDomain$1.doIntersectionPrivilege(Unknown Source)
        java.awt.EventQueue$4.run(Unknown Source)
        java.awt.EventQueue$4.run(Unknown Source)
        java.security.AccessController.doPrivileged(Native Method)
        java.security.ProtectionDomain$1.doIntersectionPrivilege(Unknown Source)
        java.awt.EventQueue.dispatchEvent(Unknown Source)
        java.awt.SequencedEvent.dispatch(Unknown Source)
        java.awt.EventQueue.dispatchEventImpl(Unknown Source)
        java.awt.EventQueue.access$400(Unknown Source)
        java.awt.EventQueue$3.run(Unknown Source)
        java.awt.EventQueue$3.run(Unknown Source)
        java.security.AccessController.doPrivileged(Native Method)
        java.security.ProtectionDomain$1.doIntersectionPrivilege(Unknown Source)
        java.security.ProtectionDomain$1.doIntersectionPrivilege(Unknown Source)
        java.awt.EventQueue$4.run(Unknown Source)
        java.awt.EventQueue$4.run(Unknown Source)
        java.security.AccessController.doPrivileged(Native Method)
        java.security.ProtectionDomain$1.doIntersectionPrivilege(Unknown Source)
        java.awt.EventQueue.dispatchEvent(Unknown Source)
        java.awt.EventDispatchThread.pumpOneEventForFilters(Unknown Source)
        java.awt.EventDispatchThread.pumpEventsForFilter(Unknown Source)
        java.awt.EventDispatchThread.pumpEventsForHierarchy(Unknown Source)
        java.awt.EventDispatchThread.pumpEvents(Unknown Source)
        java.awt.EventDispatchThread.pumpEvents(Unknown Source)
        java.awt.EventDispatchThread.run(Unknown Source)

System Information:
    Operating System: Windows 7
    Operating System Version: 6.1
    Operating System Architecture: amd64
    Java version: 1.7.0_45 64 bit
    Total Memory: 165 MB
    Max Memory: 2709 MB
    Memory Free: 134 MB
    CPU Cores: 8

**Comment by CannibalVox** [comment]
If you fabricate a bug report again I will ban you.

**Comment by Meus-Artis** [comment]
i didnt fabricate anything, i just copied and pasted my report

**Comment by CannibalVox** [comment]
Your report has the exact same memory values as your 1.8.0-ea report, and your stack trace includes methods that were added in java 1.8.  So no, you took your 1.8.0-ea report, modified the java version, and resubmitted it as new.  Incidentally, I did a google search about this issue, and many java 1.8.0-ea users are reporting it as a bug with java 1..8.0-ea.  So downgrading as you were told before will fix this issue.
